.gitignore

```diff
@@ -6,5 +6,6 @@ coverage
 npm-debug.log
 
 */**/yarn.lock
+*/**/package-lock.json
```

Comment:
The only reason I am including the root `yarn.lock` is that it triggers Travis CI to install using `yarn` which leads to faster CI cycles.

It is a pretty silly reason, if someone has a better way then I'd love to remove yarn.lock, all those versions of a 100+KB file just generate dead weight in the repository...